guide/how-it-works/ecash/fedimint.md

```diff
@@ -0,0 +1,130 @@
+
+* **High resilience** - The mint remains operational even if one or more guardians are temporarily offline, ensuring continuous access.
+
+#### Cons:
+
+* **Complex setup** - Coordinating multiple guardians adds steps and cost.
+
+
+### Federated single guardian mints
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/single-mint@1x.jpg"
+   retina = "/assets/images/guide/how-it-works/ecash/single-mint@2x.jpg"
+   mobile = "/assets/images/guide/how-it-works/ecash/single-mintmobile@2x.jpg"
+   mobileRetina = "assets/images/guide/how-it-works/ecash/single-mint-mobile@2x.jpg"
+   alt-text = "An illustration of how users, ecash, and a single guardian mint interact."
+   caption = "How a guardian interacts with users of a single guardian mint."
+   width = 800
+   height = 456
+   layout = "full-width"
+%}
+
+This model offers a compromise between security and simplicity. A single guardian manages the mint but runs multiple servers, providing some level of resilience without the need for multiple guardians.
+
+#### Pros:
+
+* **Enhanced Resilience** - Even if some servers fail, the mint remains operational, offering a good balance of security and simplicity.
+
+* **Simplified Setup** - Easier to set up compared to a fully federated mint, making it accessible while still offering improved resilience over a solo setup.
+
+#### Cons:
+
+* **Security Trade-off** - With only one guardian, there’s still a single point of failure, making it less secure than a fully federated setup.
+
+### Unfederated Mints (Solo Mints)
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/solomint@1x.jpg"
+   retina = "/assets/images/guide/how-it-works/ecash/solomint@2x.jpg"
+   mobile = "/assets/images/guide/how-it-works/ecash/solomint-mobile@2x.jpg"
+   mobileRetina = "assets/images/guide/how-it-works/ecash/solomint-mobile@2x.jpg"
+   alt-text = "An illustration of how users, ecash, and a solo mint interact."
+   caption = "How a guardian interacts with users of a solo mint."
+   width = 800
+   height = 456
+   layout = "full-width"
+%}
+
+Unfederated Mints are the simplest and most cost-effective option. Here, a single guardian manages the entire mint, making it easy to set up but with significant trade-offs in terms of security and resilience.
+
+#### Pros:
+
+* **Quick and Easy Setup** - Ideal for those who need a simple, low-cost solution.
+
+* **Cost-Effective** - Fewer resources are required, making it an attractive option for smaller operations.
+
+#### Cons:
+
+* **High Risk** - With all control in the hands of one guardian, the risks of loss or theft are much higher. Additionally, if the server goes offline, users lose access to the federation until it’s restored.
+
+## When to Use Fedimint
+* **Federated Trust Models** - Use Fedimint when your application benefits from spreading trust across multiple entities.
```

Comment:
"I don’t mean to nitpick, but I’m unsure about using the word ‘mature’ here. It might unintentionally imply that Cashu is immature. Otherwise, I think the rest is solid. What do you think?

"Use Fedimint whenever a, custodial Lightning and ecash wallet is required where single points of failure aren't acceptable, i.e. when larger amounts are being stored or reliability is a concern."

Reply:
> "Use Fedimint whenever a, custodial Lightning and ecash wallet is required where single points of failure aren't acceptable, i.e. when larger amounts are being stored or reliability is a concern."

Sounds good to me :)

> I don’t mean to nitpick, but I’m unsure about using the word ‘mature’ here.

Fine by me, maturity is a spectrum.

Cashu is more of a protocol spec after all (and that part seems solid), while Fedimint has one reference implementation by design, so comparisons are hard (I can't know all implementations). What I did hear during multiple conversations with Cashu developers at hackdays and such is that client state management is still an open issue, something we solved over a year ago on the Fedimint side. To be fair, while Fedimint is more complex by its nature we also had a head-start (first commit Jan 2021).

[Calle even tweetet about it recently](https://x.com/callebtc/status/1825907599486652924). So I don't think it's an unfair characterization from my side, but not worth fighting over.